portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -87,12 +90,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get timestamp of when comment was submitted for sorting purposes
     long timestamp = System.currentTimeMillis();
 
+    // Calculate sentiment score of user-inputted comment
+    Document doc = Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
```

Comment:
I think the Datastore is inferring the sentiment store as a double when I store the value in there because I tried using float in the model, but when I read from the datastore it would result in a java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Float.